website/docs/guides/migration/versions/02-upgrading-to-v1.5.md

```diff
@@ -56,7 +56,50 @@ models:
     tests: [] # todo! add tests later
     config: ...
 ```
-Some options that could previously be specified before a sub-command can now only be specified afterward. For example, `dbt --profiles-dir . run` isn't valid anymore, and instead, you need to use `dbt run --profiles-dir .`
+
+Some options that could previously be specified _after_ a subcommand can now only be specified _before_. This includes the inverse of the command, `--write-json` and `--no-write-json`, for example.  The list of affected commands are:
+
+<details>
+<summary>List of affected commands</summary>
```

Comment:
Could we do either "flags", "arguments", or "options" instead of "commands"? Something like this?

 ```suggestion
Some options that could previously be specified _after_ a subcommand can now only be specified _before_. This includes the inverse of the command, `--write-json` and `--no-write-json`, for example.  The list of affected options are:

<details>
<summary>List of affected options</summary>
```